src/translations/sqlb_nl.ts

```diff
@@ -0,0 +1,7344 @@
+        <source>Open preview dialog for printing displayed image</source>
+        <translation>Open voorvertoningsdialoogvenster om getoonde afbeelding af te drukken</translation>
+    </message>
+    <message>
+        <location filename="../ImageViewer.ui" line="129"/>
+        <source>Ctrl+P</source>
+        <translation></translation>
+    </message>
+</context>
+<context>
+    <name>ImportCsvDialog</name>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="14"/>
+        <source>Import CSV file</source>
+        <translation>CSV-bestand importeren</translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="25"/>
+        <source>Table na&amp;me</source>
+        <translation>&amp;Tabelnaam</translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="38"/>
+        <source>&amp;Column names in first line</source>
+        <translation>&amp;Kolomnamen op eerste regel</translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="55"/>
+        <source>Field &amp;separator</source>
+        <translation>&amp;Veldscheidingsteken</translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="68"/>
+        <source>,</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="73"/>
+        <source>;</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="78"/>
+        <location filename="../ImportCsvDialog.cpp" line="779"/>
+        <source>Tab</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="83"/>
+        <source>|</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="88"/>
+        <location filename="../ImportCsvDialog.ui" line="158"/>
+        <source>Other (printable)</source>
+        <translation>Anders (afdrukbaar)</translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="93"/>
+        <location filename="../ImportCsvDialog.ui" line="163"/>
+        <source>Other (code)</source>
+        <translation>Anders (code)</translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="130"/>
+        <source>&amp;Quote character</source>
+        <translation>&amp;Scheidingsteken tekenreeks</translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="143"/>
+        <source>&quot;</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="148"/>
+        <source>&apos;</source>
+        <translation></translation>
```

Comment:
I think it's a symbol that doesn't need translation, so I think you've left the translation blank.
I also have a similar case in the past, but there was an comments that it is safer to copy the original text for symbol
that do not require translation. Please refer to [here](https://github.com/sqlitebrowser/sqlitebrowser/pull/2329#issuecomment-658202890) for details.